climada/entity/exposures/crop_production.py

```diff
@@ -541,62 +579,78 @@ def normalize_with_fao_cropyield(exp_firr, exp_noirr, input_dir=INPUT_DIR,
+            ratio[country] = fao_crop_production[country] / exp_tot_production[country]
 
         exp_firr_norm.value[exp_firr.region_id == iso_nr] = ratio[country] * \
         exp_firr.value[exp_firr.region_id == iso_nr]
         exp_noirr_norm.value[exp_firr.region_id == iso_nr] = ratio[country] * \
         exp_noirr.value[exp_noirr.region_id == iso_nr]
 
-        if unit == 'USD':
-            exp_noirr['value_tonnes'] = exp_noirr['value']
+        if unit == 'USD' or exp_noirr.value_unit == 'USD / y':
             exp_noirr.set_to_usd(input_dir=input_dir)
-            exp_firr['value_tonnes'] = exp_firr['value']
+        if unit == 'USD' or exp_firr.value_unit == 'USD / y':
             exp_firr.set_to_usd(input_dir=input_dir)
 
-    if returns == 'all':
-        return country_list, ratio, exp_firr_norm, exp_noirr_norm, fao_yield, exp_totyield
+    exp_firr_norm.tag.description = exp_firr_norm.tag.description+' normalized'
+    exp_noirr_norm.tag.description = exp_noirr_norm.tag.description+' normalized'
+
+    if return_data:
+        return country_list, ratio, exp_firr_norm, exp_noirr_norm, \
+            fao_crop_production, exp_tot_production
     return country_list, ratio, exp_firr_norm, exp_noirr_norm
 
 def normalize_several_exp(input_dir=INPUT_DIR, output_dir=OUTPUT_DIR,
                           yearrange=np.array([2008, 2018]),
-                          unit='t', returns='all'):
+                          unit='t', return_data=True):
     """
-
-        Parameters:
+        Optional Parameters:
         input_dir (str): directory containing exposure input data
         output_dir (str): directory containing exposure datasets (output of exposure creation)
-        yearrange (array): the mean yield in this year range is used to normalize the exposure
-            data (default 2008-2018)
+        yearrange (array): the mean crop production in this year range is used to normalize
+            the exposure data (default 2008-2018)
         unit (str): unit in which to return exposure (t/y or USD/y)
-        returns (str): returned output
-            'all': lists containing data for each exposure file. Lists: crops, country list,
-            ratio = FAO/ISIMIP, normalized exposures, yield per country as documented by the
-            FAO and calculated by the ISIMIP dataset
-            else: lists containing data for each exposure file. Lists: crops, country list,
+        return_data (str): returned output
+            True: lists containing data for each exposure file. Lists: crops, country list,
+            ratio = FAO/ISIMIP, normalized exposures, crop production per country as documented
+            by the FAO and calculated by the ISIMIP dataset
+            False: lists containing data for each exposure file. Lists: crops, country list,
             ratio = FAO/ISIMIP, normalized exposures
-        Returns:
 
+        Returns:
+            c.f. normalize_with_fao_cp
```

Comment:
What does c.f. normalize_with_fao_cp mean?

Reply:
I think it refers to the returns of the method "normalise_with_fao_cp" (as they are similar). However, they are not exactly the same, so I adapted this docstring.